dev-haskell/unordered-containers/unordered-containers-0.2.13.0.ebuild

```diff
@@ -6,6 +6,9 @@ EAPI=7
 # ebuild generated by hackport 0.6.7.9999
 
 CABAL_FEATURES="lib profile haddock hoogle hscolour test-suite"
+# break circular dependencies:
+# https://github.com/gentoo-haskell/gentoo-haskell/issues/810
```

Comment:
Ideally the comment should show exact dependency circle. Maybe there is a better place to break the circle.

Reply:
Fixed in latest commit.